### ISSUE TITLE
Crash when selecting text

### ISSUE THREAD

**Issue by TBog** [TBog/TBLauncher#327]
# Description #
Crash reported on Playstore

```
Exception android.content.res.Resources$NotFoundException: Resource ID #0x0
  at android.content.res.ResourcesImpl.getValue (ResourcesImpl.java:255)
  at android.content.res.Resources.getDrawable (Resources.java:833)
  at android.content.Context.getDrawable (Context.java:529)
  at android.widget.Editor$SelectionModifierCursorController.initDrawables (Editor.java:5299)
  at android.widget.Editor$SelectionModifierCursorController.show (Editor.java:5293)
  at android.widget.Editor$SelectionModifierCursorController.enterDrag (Editor.java:5334)
  at android.widget.Editor.selectCurrentWordAndStartDrag (Editor.java:2019)
  at android.widget.Editor.performLongClick (Editor.java:1149)
  at android.widget.TextView.performLongClick (TextView.java:9805)
  at android.view.View.performLongClick (View.java:5703)
  at android.view.View$CheckForLongPress.run (View.java:22448)
  at android.os.Handler.handleCallback (Handler.java:761)
  at android.os.Handler.dispatchMessage (Handler.java:98)
  at android.os.Looper.loop (Looper.java:156)
  at android.app.ActivityThread.main (ActivityThread.java:6617)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:942)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:832)
```

## Device info ##

  * Device: Unknown
  * OS: Android 7.0 (SDK 24)
  * TinyBit Launcher version: 37 (6.15)

**Comment by TBog** [comment]
relevant code: https://android.googlesource.com/platform/frameworks/base/+/refs/tags/android-7.0.0_r36/core/java/android/widget/Editor.java

because `android.widget.Editor$SelectionModifierCursorController` is accessing `mTextView.mTextSelectHandleLeftRes` some changes to `Utilities.setTextSelectHandle` from `app/src/main/java/rocks/tbog/tblauncher/utils/Utilities.java` need to be made for this to work

**Comment by TBog** [comment]
should be fixed in release [v7.2](https://github.com/TBog/TBLauncher/releases/tag/v7.2)
